engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthTest.java

```diff
@@ -19,13 +19,13 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class disable OpenGL's depth test, allowing the processing of fragments
- * that would normally fail the test and would therefore be discarded.
+ * Disables depth testing in OpenGL pipeline.
  *
- * Notice that by default OpenGL has depth testing disabled but in Terasology's defaults it is enabled,
- * as depth testing is used by many nodes. It's important then to use this StateChange so that each
- * node requiring no depth testing disables it and re-enables it after the process() method has been
- * executed.
+ * OpenGL Default: DisableDepthTest
+ * Terasology Default: EnableDepthTest
+ *
+ * Type: Reset
+ * Corresponding Set: DisableDepthTest
```

Comment:
This javadoc highlight the difficulties with terse javadocs: it's easy to get confused. 

Specifically in this case the "type" of this state change is actually "set" as it sets the depth testing as disabled. The "reset" state change resets it to its (Terasology) default - depth testing is enabled. 

So, I'd write:

```
/**
* Disables OpenGL depth testing.
*
* Notice that OpenGL has depth testing disabled by default. Terasology however enables it by default as depth 
* testing is used in many nodes. It's important then to use this StateChange if a node requires the OpenGL default.
*/
```